backend/lcfs/web/api/transfer/services.py

```diff
@@ -264,37 +264,67 @@ async def update_transfer(self, transfer_data: TransferCreateSchema) -> Transfer
         # Finally, update the transfer's status and save the changes
         transfer.current_status = new_status
         transfer_result = await self.repo.update_transfer(transfer)
-        await self._perform_notificaiton_call(transfer_result)
+        await self._perform_notificaiton_call(
+            transfer,
+            status=(
+                new_status.status
+                if status_has_changed or re_recommended
+                else "Return to analyst"
+            ),
+        )
         return transfer_result
 
-    async def _perform_notificaiton_call(self, transfer):
+    async def _perform_notificaiton_call(
```

Comment:
suggestion: Fix spelling of notification

Reply:
fixed. :)